Frontend/node/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "atlas-ui",
+  "version": "1.7.10",
+  "description": "Atlas UI",
+  "private": true,
+  "license": "ISC",
+  "dependencies": {
+    "core-js": "^2.6.11",
+    "moment": "^2.27.0",
+    "presto-ui": "git+https://github.com/juspay/presto-ui.git#master",
+    "regenerator-runtime": "^0.13.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.7.7",
+    "@babel/preset-env": "^7.7.7",
+    "babel-loader": "^8.0.6",
+    "filemanager-webpack-plugin": "^2.0.5",
+    "webpack": "^4.41.5",
+    "webpack-cli": "^3.3.10",
+    "webpack-dev-server": "^3.10.1",
+    "webpack-merge": "^5.8.0"
```

Comment:
Are these dependencies used only in `watch.js`? I'm wondering if we can make watch-dev-server a self-contained sub-directory along with the Nix function.